sdk/storage/storage-blob/test/blobclient.spec.ts

```diff
@@ -1541,8 +1577,13 @@ describe("BlobClient - Object Replication", () => {
     if (!isNode) {
       this.skip();
     }
-    recorder.skip("node", "Temp file - recorder doesn't support saving the file");
-    const srcDownloadedFilePath = recorder.getUniqueName("srcdownloadedfile");
+    if (isNode && !isLiveMode()) {
+      this.skip();
+    }
```

Comment:
NIT

```
if(!isLiveMode() || !isNode){
   this.skip();
}
```

Reply:
Maybe I misunderstood how `recorder.skip` worked, but I thought `recorder.skip("node", msg)` meant "only skip recording/playback on node" - is that not the case?

Reply:
No, you had the right thing, your understanding is also correct.

I'm just merging the two `if` checks into one.

```js
    if (!isNode) {
      this.skip();
    }
    if (isNode && !isLiveMode()) {
      this.skip();
    }
```

->

```js
   if(!isLiveMode() || !isNode){
      this.skip();
   }
```

Reply:
oh! I see now! Thanks!